osbs/cli/main.py

```diff
@@ -457,6 +458,11 @@ def str_on_2_unicode_on_3(s):
 
 
 def cli():
+    try:
+        version = pkg_resources.get_distribution("osbs-client").version
+    except pkg_resources.DistributionNotFound:
```

Comment:
In what scenario would this exception be triggered? When I have it installed from git, `get_distribution` is still able to find it properly.